dbmigrator/__init__.py

```diff
@@ -12,4 +13,12 @@
 logger.addHandler(handler)
 
 
-__all__ = ('logger',)
+def deferrable(func):
+    """A decorator to mark the migration as deferred by default.  The state of
+    the migration can be changed later by using `mark`.
+    """
```

Comment:
ID go with `@deferred`, since it sets the actual state, not give it the property of being allowed to be deferred.